ts/Core/Series/SeriesDefaults.ts

```diff
@@ -1653,13 +1653,9 @@ const seriesDefaults: PlotOptionsOf<Series> = {
          * series animation has finished. Setting to `false` renders the
          * data label immediately. If set to `true` inherits the defer
          * time set in [plotOptions.series.animation](#plotOptions.series.animation).
-         * If set to a number, a defer time is specified in milliseconds.
-         *
-         * @sample highcharts/plotoptions/animation-defer
-         *         Set defer time
          *
          * @since     4.0.0
-         * @type      {boolean|number}
+         * @type      {boolean}
```

Comment:
Do you have a jsFiddle demo where it doesn't work?

Reply:
I think here: https://jsfiddle.net/BlackLabel/19cysqpx/

Plus if you take a look at the `interface DataLabelOptions` in `DataLabelOptions.d.ts` its type is `boolean`, not `boolean | number`.

Reply:
FYI @TorsteinHonsi